test/utils/describeConformance.js

```diff
@@ -148,28 +148,31 @@ export function describeRef(element, getOptions) {
  */
 export function testRootClass(element, getOptions) {
   it('applies the root class to the root component if it has this class', () => {
-    const { classes, mount } = getOptions();
+    const { classes, render } = getOptions();
     if (classes.root == null) {
       return;
     }
 
     const className = randomStringValue();
-    let wrapper = mount(React.cloneElement(element, { className }));
+    const { container, setProps } = render(React.cloneElement(element, { className }));
+
+    // The component is portaled
```

Comment:
I will reverse the approach, this sucks

Reply:
I don't understand why this change is required?

Reply:
I'm on it. A new commit is coming.

Reply:
900a368f09

Reply:
The change was required because with react-testing-library, we have used `container.firstChild` so far to determine the "root element". This strategy fails with portals.